src/main/java/com/google/codeu/data/User.java

```diff
@@ -1,12 +1,24 @@
 package com.google.codeu.data;
 
+import java.util.ArrayList;
+
 public class User {
 
   private String email, aboutMe;
+  private ArrayList<String> mentions;
```

Comment:
This is a String instead of UUID because Datastore's setProperty function can't handle UUID; I think if we want to do anything going forward that requires UUID, we can use fromString().

Reply:
Just to clarify, these are Message UUID Strings?

Reply:
nit: you should declare a List<String> mentions; and when you actually initialize/supply the mentions that's when you use an ArrayList. This helps support other implementations of Lists in future Java versions, etc.

Reply:
Yes; whenever/if we decide to implement something that allows people to see what Users have been mentioned in, we'll have to Message.getId().toString() (UUID -> String) then put into mentions (unless there's an easy way to store UUID in Datastore as a property?)

What would be good? Something like mentionedMessageId? Seems a little hard to read but more descriptive

Reply:
I'll change it to List!